CHANGELOG.md

```diff
@@ -4,16 +4,28 @@ All notable changes to this project will be documented in this file.
 <!-- markdownlint-disable MD024 -->
 
 ## [Unreleased]
+
+## [34.0.0] - 2024-08-30
+
 <!-- None < 2024-08-01 11:45:37+00:00 -->
```

Comment:
That None smells like a tool bug :)

Reply:
nope, it's simply because the release does not exist yet, will be auto-fixed on next run